client/scripts/views/components/navigation/index.ts

```diff
@@ -193,6 +195,10 @@ const Navigation: m.Component<{ activeTag: string }, { communitySwitcherVisible:
               onclick: (e) => m.route.set(`/${app.activeId()}/members/`),
               contentLeft: m(Icon, { name: Icons.BOX }),
             }),
+          // admin panel (all communities)
+          isRoleOfCommunity(app.vm.activeAccount, app.login.addresses, app.login.roles, 'admin', app.activeId()) &&
+          (app.community || app.chain) &&
+          m(AdminPanel),
```

Comment:
I believe this won't lint correctly - what are you using for text editor/do you have on-the-fly linting enabled?

(this shouldn't block the PR, just checking)

Reply:
i use vscode with on the fly linting. The `&&`s had a little red underneath them but I didn't notice. I fixed this